src/world/world.c

```diff
@@ -657,5 +657,5 @@ Area gAreas[] = {
     AREA(mgm, "ミニゲーム"),  // minige-mu [Minigames]
     AREA(gv, "ゲームオーバー"),  // ge-mu o-ba- [Game Over]
     AREA(tst, "テストマップ"),  // tesuto mappu [Test map]
-    0, 0, 0, 0,
+    {},
```

Comment:
imo, these should remain as properly zero'd (C doesn't guarantee what these get initialized to; even if they do in this case, could cause problems later with e.g. more modern compilers)

Reply:
@ethteck thoughts?

Reply:
https://stackoverflow.com/questions/11152160/initializing-a-struct-to-0 From this discussion, it seems this isn't really a big issue, at least with GCC, which is the only compiler I see us caring about in the near future. If it becomes an issue later on, we can zero-initialize everything, but until then this is easy to search for if we need to change it and looks a lot cleaner.